L1Trigger/Phase2L1ParticleFlow/interface/conifer.h

```diff
@@ -1,5 +1,5 @@
-#ifndef CONIFER_CPP_H__
-#define CONIFER_CPP_H__
+#ifndef L1TRIGGER_PHASE2L1PARTICLEFLOW_H
+#define L1TRIGGER_PHASE2L1PARTICLEFLOW_H
```

Comment:
no wait this change is wrong, as the name should be unique.
It should be something that has both the subsystem and the file, e.g. `L1Trigger_Phase2L1ParticleFlow_Conifer_h`

Reply:
Ah, this is my naming error, That's a quick fix.